### PR TITLE
Removes meat subtype from sheetifier

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Removes meat subtype from the list of items the sheetifier accepts.

## How This Contributes To The Skyrat Roleplay Experience

You're not supposed to be able to do that on SR.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl: LT3
code: Sheetifier no longer accepts meat products
/:cl:

### CODE REVIEW

**Review by tf-4 — CHANGES_REQUESTED** [review]
not really any reason to move this whole proc to a new file, just comment out the addcomponent and add a new, modified one with a `//skyrat edit` comment.

**Comment by lessthnthree** [comment]
> not really any reason to move this whole proc to a new file, just comment out the addcomponent and add a new, modified one with a `//skyrat edit` comment.
Okay

File: code/game/machinery/sheetifier.dm [hunk 1]
```diff
@@ -10,8 +10,8 @@
...
 	. = ..()
-
-	AddComponent(/datum/component/material_container, list(/datum/material/meat, /datum/material/hauntium), MINERAL_MATERIAL_AMOUNT * MAX_STACK_SIZE * 2, MATCONTAINER_EXAMINE|BREAKDOWN_FLAGS_SHEETIFIER, typesof(/datum/material/meat) + /datum/material/hauntium, list(/obj/item/food/meat, /obj/item/photo), null, CALLBACK(src, PROC_REF(CanInsertMaterials)), CALLBACK(src, PROC_REF(AfterInsertMaterials)))
+//	AddComponent(/datum/component/material_container, list(/datum/material/meat, /datum/material/hauntium), MINERAL_MATERIAL_AMOUNT * MAX_STACK_SIZE * 2, MATCONTAINER_EXAMINE|BREAKDOWN_FLAGS_SHEETIFIER, typesof(/datum/material/meat) + /datum/material/hauntium, list(/obj/item/food/meat, /obj/item/photo), null, CALLBACK(src, PROC_REF(CanInsertMaterials)), CALLBACK(src, PROC_REF(AfterInsertMaterials))) // SKYRAT EDIT CHANGE
+	AddComponent(/datum/component/material_container, list(/datum/material/hauntium), MINERAL_MATERIAL_AMOUNT * MAX_STACK_SIZE * 2, MATCONTAINER_EXAMINE|BREAKDOWN_FLAGS_SHEETIFIER, /datum/material/hauntium, list(/obj/item/photo), null, CALLBACK(src, PROC_REF(CanInsertMaterials)), CALLBACK(src, PROC_REF(AfterInsertMaterials))) // SKYRAT EDIT CHANGE
```
Comment (tf-4) [inline code/game/machinery/sheetifier.dm]:
```suggestion
	AddComponent(/datum/component/material_container, list(/datum/material/hauntium), MINERAL_MATERIAL_AMOUNT * MAX_STACK_SIZE * 2, MATCONTAINER_EXAMINE|BREAKDOWN_FLAGS_SHEETIFIER, /datum/material/hauntium, list(/obj/item/photo), null, CALLBACK(src, PROC_REF(CanInsertMaterials)), CALLBACK(src, PROC_REF(AfterInsertMaterials))) // SKYRAT EDIT CHANGE - REMOVES MEAT FROM SHEETIFIER
```

**Comment by Iamgoofball** [comment]
>You're not supposed to be able to do that on SR.

Why?

**Review by tf-4 — APPROVED** [review]
restoring an older edit that got overwritten

**Comment by Iamgoofball** [comment]
I know that, but like, why?

**Pull request merged**